src/test/java/api/items/ItemApiMoveExamples.java

```diff
@@ -0,0 +1,148 @@
+package api.items;
+
+import static api.support.InstanceSamples.smallAngryPlanet;
+import static org.folio.inventory.support.http.ContentType.APPLICATION_JSON;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import java.net.MalformedURLException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.folio.inventory.domain.items.ItemStatusName;
+import org.folio.inventory.support.http.client.IndividualResource;
+import org.folio.inventory.support.http.client.Response;
+import org.folio.inventory.support.http.client.ResponseHandler;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import api.support.ApiRoot;
+import api.support.ApiTests;
+import api.support.InstanceApiClient;
+import api.support.builders.HoldingRequestBuilder;
+import api.support.builders.ItemRequestBuilder;
+import api.support.builders.ItemsMoveRequestBuilder;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import junitparams.JUnitParamsRunner;
+
+@RunWith(JUnitParamsRunner.class)
+public class ItemApiMoveExamples extends ApiTests {
+
+  private static final String HOLDINGS_RECORD_ID = "holdingsRecordId";
+
+  @Test
+  public void canCreateItemsMove() throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
+
+    UUID instanceId = UUID.randomUUID();
+    InstanceApiClient.createInstance(okapiClient, smallAngryPlanet(instanceId));
+
+    final UUID existedHoldingId = createHoldingForInstance(instanceId);
+    final UUID newHoldingId = createHoldingForInstance(instanceId);
+
+    Assert.assertNotEquals(existedHoldingId, newHoldingId);
+
+    final IndividualResource createItem1 = itemsClient.create(new ItemRequestBuilder().forHolding(existedHoldingId)
+      .withBarcode("645398607547")
+      .withStatus(ItemStatusName.AVAILABLE.value()));
+
+    final IndividualResource createItem2 = itemsClient.create(new ItemRequestBuilder().forHolding(existedHoldingId)
+      .withBarcode("645398607546")
+      .withStatus(ItemStatusName.AVAILABLE.value()));
+
+    JsonObject itemsMoveRequestBody = new ItemsMoveRequestBuilder(newHoldingId,
+        new JsonArray(Arrays.asList(createItem1.getId(), createItem2.getId()))).create();
+
+    CompletableFuture<Response> postItemsMoveCompleted = new CompletableFuture<>();
+    okapiClient.post(ApiRoot.move(), itemsMoveRequestBody, ResponseHandler.any(postItemsMoveCompleted));
+    Response postItemsMoveResponse = postItemsMoveCompleted.get(5, TimeUnit.SECONDS);
+
+    assertThat(postItemsMoveResponse.getStatusCode(), is(201));
```

Comment:
What new records or resource are being created?

If there aren't any, why does this respond with a CREATED status rather than OK?

Reply:
@marcjohnson-kint so, this is not clear. From the one side this is POST with creation of some kind "virtual move" of items. So 201 is more appropriate from my perspective. 200 looks useful for GET some resource. But here we don't need to retrieve any resource in the happy path. So it's up to you. Let me know if I need to change this code.

Reply:
> From the one side this is POST with creation of some kind "virtual move" of items. So 201 is more appropriate from my perspective. 200 looks useful for GET some resource.

I afraid I don't really follow how any resources are created. If a resource was created to represent the move itself, I could understand using 201 CREATED rather than 200 OK. Maybe that is what you mean by `virtual move`?